TenElephants/TenElephants/Views/RoundButtonWithBlur.swift

```diff
@@ -8,22 +8,34 @@
 import Foundation
 import UIKit
 
-final class CloseButton: UIButton {
+final class RoundButtonWithBlur: UIButton {
+    enum buttonType {
```

Comment:
Типы с заглавной буквы
```suggestion
    enum ButtonType {
```